tethys_gizmos/gizmo_options/jobs_table.py

```diff
@@ -437,6 +439,11 @@ def __init__(
         if self.show_actions:
             self.num_cols += 1
 
+        # Derive base_ajax_url dynamically for use in JavaScript
+        # e.g.: /developer/gizmos/ajax/9999/action/delete
+        self.base_ajax_url = reverse("gizmos:delete_job", kwargs={"job_id": 9999})
+        self.base_ajax_url = self.base_ajax_url.replace("9999/action/delete", "")
```

Comment:
This is an interesting way to do this. It feels like a work around. Any thoughts on a more direct way to get a base URL? We may need to do similar things in other places.

Reply:
It's a workaround only because we don't have a URL that resolves to the pattern we needed for our template URL. Maybe we should add a URL for `/developer/gizmos/ajax/`? Not sure what the view would render as, so it would be a different hacky workaround. Either way, I think using `reverse()` in controllers or `url` in templates is what we should do to get URLs.